projects/ltc2672_iio/app/ltc2672_iio.c

```diff
@@ -80,6 +82,10 @@ static int ltc2672_iio_attr_available_set(void *device,
 /*	Number of IIO devices */
 #define NUM_OF_IIO_DEVICES	1
 
+/* Device names */
+#define DEVICE_LTC2662_16	 "ltc2662-16"
+#define DEVICE_LTC2672_16	 "ltc2672-16"
+
```

Comment:
We already have active device name macros defined in app_config.h file (may be just for DC2903A).. shall we move these there or try to use them as needed?